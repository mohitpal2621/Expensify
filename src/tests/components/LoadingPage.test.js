import React from "react";
import { render } from "@testing-library/react";
import LoadingPage from "../../components/LoadingPage";

test("Should render the loading page", () => {
    const {container} = render(<LoadingPage/>);
    expect(container).toMatchSnapshot();

})