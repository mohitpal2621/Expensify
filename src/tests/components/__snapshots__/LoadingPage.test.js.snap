// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the loading page 1`] = `
<div>
  <div
    class="loader"
  >
    <img
      alt="Loading..."
      class="loader__image"
      src="/images/loader.gif"
    />
  </div>
</div>
`;
